README.md

```diff
@@ -83,6 +83,35 @@ val books: List<Book> = queryFactory.listQuery {
 }
 ```
 
+#### with nullable condition
+
```

Comment:
안녕하세요. :)
해당 documentation 을 조금 더 자세하게 작성해주시면 좋겠습니다. 왜냐하면 where PredicateSpec 에 대하여 null 을 허용되는 스펙으로 되었기 때문인데요.

where 의 PredicateSpec 을 nullable 로 바꾸게 되면 말씀주신 것처럼 nullable 한 검색조건에 대하여 
코드를 조금 더 가독성 있게 작성할 수 있다는 장점을 얻었지만, 
반대로 잃었던 단점은 where 조건에 대하여 개발자가 empty spec이 들어갈 수 있다라는 컴파일 타임에 인지할 수 있는 상황이 사라졌다고 생각합니다.

이는 충분한 테스트와 코드 리뷰를 통해서 발견할 수도 있지만, 라이브러리를 제공하는 사람의 입장에서도 어느정도 주의점으로 명시할 정도라고 생각되기도 하는데요. 
`설명` 과 `주의점` 에 대하여 작성해주셔도 괜찮을 것 같은 의견인데, 어떠신가요?

-- english
Hello. :)
I would appreciate it if you could write the documentation in more detail. Because null is allowed spec for where Predict Spec.

If you change the predicate spec to nullable, you can write the code more legibly for null search conditions, as you mentioned
On the contrary, I think the disadvantage of losing is that the situation that developers can recognize about where conditions can be filled with empty spec has disappeared.

This can be found through sufficient testing and code review, but I think it's a caution point for the library provider.
I think it would be okay if you write down 'Explanation' and 'Caution', what do you think?

Reply:
@huisam 좋은 리뷰 감사합니다!

___

#### with nullable condition
`null` in `where condition` converts to `EmptyPredicateSpec` object.

_note that `EmptyPredicateSpec` turns into `1=1` sql_

```kotlin
val books: List<Book> = queryFactory.listQuery {
        select(entity(Book::class))
        from(entity(Book::class))
        where(
            name?.run { column(Book::author).equal(this) }
        )
    }
```
___

이렇게 작성해보았는데 괜찮을까요?

Reply:
-- korean
네 저는 충분하다고 생각합니다.

--
english

Yes, I think that's enough.

Reply:
@huisam 
[docs: add descriptions for where method](https://github.com/line/kotlin-jdsl/pull/75/commits/fbc308100a8767163652c000e9f61129a34fc22a)
말씀하신 null condition에 대한 설명을 적은 커밋입니다.
또한 whereAnd와 whereOr에 대한 설명도 필요하다고 생각해서 적어보았는데 괜찮은지 피드백 부탁드리겠습니다 감사합니다!